connector/src/main/resources/org/jboss/as/connector/subsystems/resourceadapters/LocalDescriptions.properties

```diff
@@ -67,56 +67,56 @@ connection-definitions.clear-statistics=Clear statistics values for this resourc
-connection-definitions.security-domain-and-application=Indicates that either app supplied parameters (such as from getConnection(user, pw)) or Subject (from security domain) are used to distinguish connections in the pool.
+connection-definitions.same-rm-override=Using this attribute, you can unconditionally set whether javax.transaction.xa.XAResource.isSameRM(XAResource) returns true or false.
+connection-definitions.security-application=Indicates that application-supplied parameters, such as from getConnection(user, pw), are used to distinguish connections in the pool.
+connection-definitions.security-domain=Specifies the security domain which defines the javax.security.auth.Subject that is used to distinguish connections in the pool.
+connection-definitions.security-domain-and-application=Indicates that either application-supplied parameters, such as from getConnection(user, pw), or Subject (from security domain), are used to distinguish connections in the pool.
 connection-definitions.set-tx-query-timeout=Whether to set the query timeout based on the time remaining until transaction timeout. Any configured query timeout will be used if there is no transaction.
 connection-definitions.test-connection-in-pool=Test if a connection can be obtained.
 connection-definitions.use-ccm=Enable the use of a cached connection manager.
-connection-definitions.sharable=Enable the use of sharable connections which allows lazy association to be enabled if supported
+connection-definitions.sharable=Enable the use of sharable connections, which allows lazy association to be enabled if supported.
 connection-definitions.enlistment=Defines if lazy enlistment should be used if supported by the resource adapter.
 connection-definitions.use-fast-fail=Whether to fail a connection allocation on the first try if it is invalid (true) or keep trying until the pool is exhausted of all potential connections (false).
 connection-definitions.use-java-context=Setting this to false will bind the object into global JNDI.
 connection-definitions.use-try-lock=Any configured timeout for internal locks on the resource adapter objects in seconds.
-connection-definitions.wrap-xa-resource=Should the XAResource instances be wrapped in an org.jboss.tm.XAResourceWrapper instance.
-connection-definitions.xa-resource-timeout=The value is passed to XAResource.setTransactionTimeout(), in seconds. Default is zero.
-connection-definitions.validate-on-match=The validate-on-match element specifies if connection validation should be done when a connection factory attempts to match a managed connection. This is typically exclusive to the use of background validation
+connection-definitions.wrap-xa-resource=Specifies whether XAResource instances should be wrapped in an org.jboss.tm.XAResourceWrapper instance.
+connection-definitions.xa-resource-timeout=The value is passed to XAResource.setTransactionTimeout(), in seconds. The default is 0.
```

Comment:
The default values should be obtained from the model itself, not in the description as they tend to go out of sync. We should rather remove the sentence with the default value.